integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -571,4 +572,22 @@ class CarbonFileMetastore extends CarbonMetaStore {
       case _ => throw new NoSuchTableException(tableIdentifier.database.get, tableIdentifier.table)
     }
   }
+
+  override def getTableInfo(identifier: AbsoluteTableIdentifier)
```

Comment:
All ready API exists in SchemReader , better use from there. As this API is only for filebased schema